Adding capacitor/cli brings in @types/node

Description of the problem:

When adding Capacitor to a project of ours, the Typescript compiler starts complaining about `error TS2300: Duplicate identifier 'require'.` and `error TS2554: Expected 1 arguments, but got 2.` etc.

(We use AMD for module loading in this project.)

Affected platform
- [X] Android
- [X] iOS
- [X] electron
- [X] web

OS of the development machine
- [ ] Windows
- [X] macOS
- [ ] linux

Other information:
The problem is probably that `capacitor/cli/package.json` lists `@types/tar"` in its `dependencies` section and not in the `devDependencies` section.

Capacitor version:
1.0.0-beta.19

node version:
v10.15.2

npm version:
6.4.1

CocoaPods version:


Steps to reproduce:




Link to sample project:

--- a/cli/package.json
+++ b/cli/package.json
@@ -44,7 +44,6 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@types/tar": "^4.0.0",
     "chalk": "^2.3.0",
     "commander": "^2.12.2",
     "compare-versions": "^3.1.0",
@@ -68,6 +67,7 @@
     "@types/open": "^6.1.0",
     "@types/ora": "^1.3.1",
     "@types/semver": "^5.4.0",
+    "@types/tar": "^4.0.0",
     "@types/which": "^1.0.28",
     "@types/xml2js": "^0.4.2",
     "jest": "^23.6.0",

